modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/Deployment.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.deployment;
```

Comment:
i feel it is a little confusing that we have two modules - one called deployment and other one called terraform-deployment. Would it be possible to move terraform-deployment under 'deployment' package itself?
for example -  deployment -> deployers->  terraform, pulumi, etc (all deployer types will get a new folder here).

Reply:
agree.